src/Kestrel.Transport.Sockets/Internal/SocketTrace.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal
+{
+    public class SocketTrace : ISocketTrace
```

Comment:
Isn't this exactly (or almost exactly) the same as `LibuvTrace`?

Reply:
Yes. Minus a few things.

Reply:
Then how about having a common base class in `Transport.Abstractions`? Do you think that might not suit future transports?

Reply:
It could be a good idea, but it's not 100% clear exactly which log methods belong in the base class. Some transports might not be TCP-based and not have a concept of things like sending vs receiving a FIN.

I created #2101 to discuss this idea further.